Pipfile

```diff
@@ -0,0 +1,28 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
+wagtail = "==2.5.1"
+django-cors-headers = "==2.2.0"
+django-extensions = "==2.0.7"
+django-filter = "==2.1.0"
+django-modeltranslation = "==0.13.1"
+wagtail-modeltranslation = "==0.10.1"
+dj-database-url = "==0.5.0"
+graphene-django = "==2.2.0"
+gunicorn = "==19.8.1"
+heroku3 = "==3.3.0"
+psycopg2-binary = "==2.7.4"
+django-webpack-loader = "==0.6.0"
+boto3 = "==1.9.94"
+django-storages = "==1.6.6"
+django-dbbackup = "==3.2.0"
+django-smuggler = "==0.9.1"
+PyYAML = ">=4.2b1"
+
+[requires]
+python_version = "3.7"
```

Comment:
I'm down for this(and its the version i use locally), but FYI we had python version pinned to 3.6 in our Dockerfiles

Reply:
Good call. I'd vote in favor of us moving Joplin to the latest version of Joplin.

Reply:
@briaguya should we just up the python version in our docker containers and see if we notice anything breaking or hold off for now?